ta4j-core/src/main/java/org/ta4j/core/num/PrecisionNum.java

```diff
@@ -192,6 +192,9 @@ public static PrecisionNum valueOf(long val) {
     /**
      * Returns a {@code Num} version of the given {@code float}.
      *
+     * <p>
```

Comment:
Please apply correct formatting: `<p>` is not necessary. Please also apply asterisks in a convenient way (in convention applied in other javadocs)